docs/zenith/developer/python/539756-advanced-programming.md

```diff
@@ -96,6 +97,22 @@ The result will be:
 }
 ```
 
+## Write an async module constructor function
+
+Sometimes it's convenient to have a special function for constructing the main module object. This can, for example, enable use of async calls while initializing the object.
```

Comment:
```suggestion
Sometimes it's convenient to have a special function for constructing the main module object. This can, for example, enable use of asynchronous calls while initializing the object.
```